src/grain/distribution.js

```diff
@@ -81,7 +81,7 @@ export type CredHistory = $ReadOnlyArray<CredTimeSlice>;
 export function distribution(
   strategy: DistributionStrategy,
   credHistory: CredHistory,
-  earnings: Map<NodeAddressT, Grain>,
+  lifetimeEarningsMap: Map<NodeAddressT, Grain>,
```

Comment:
nit: would be nice to update the comment (line 78) to explain a bit more what this variable is